### ISSUE TITLE
Windows 10 in VMWare created with too-low of a machine version to boot in VMWare Fusion

### ISSUE THREAD

**Issue by zestysoft** [chef/bento#1271]
When creating the vmware virtual machine based on the windows-10.json file, Windows attempts to boot but hangs at windows logo.
# Describe the problem

Fully boot to allow winrm code to run
## Software Version

git clone master, on MacOS Catalina for VMWare Fusion 11 Pro
## Replication Case

packer build -var headless=false --only=vmware-iso windows-10.json
## Stacktrace

<details><summary>  </summary>
```
```
</details>

## Possible Solution

If I stop the vmware instance and edit the vmx file to bump up the virtual hardware version to 10 it works correctly:
virtualhw.version = "10"

If you can tell me where to make this change, I'll submit a PR.

edit: When I tried windows-10.json with:
30ad1cdf5d0670f12788005131e24862f6ab8aab  SW_DVD5_WIN_ENT_10_64BIT_English_MLF_X20-26061.ISO

It did not need this adjustment, so there must be something about the ent eval iso that has different hardware requirements?

**Comment by zestysoft** [comment]
Okay, I can confirm this is due to newer hardware requirements for later versions of Windows.  SW_DVD5_WIN_ENT_10_64BIT_English_MLF_X20-26061.ISO is the original Release To Manufacturer (RTM) version of Windows 10 Enterprise (1507 Build 10240.16384).  After upgrading to 1909, I could no longer boot into windows until I upgraded the virtual hardware.

**Comment by tas50** [comment]
Seems like setting things to 11 is probably pretty reasonble. I'll test that out today. Thanks for such a detailed issue.

**Comment by tas50** [comment]
@zestysoft Any chance you can try out https://app.vagrantup.com/tas50/boxes/windows_10 which was built at 11.

**Comment by zestysoft** [comment]
Well, I got a "Timed out while waiting for the machine to boot" however when I opened the resultant .vmx file to re-attach the headless vm in vmware fusion, it opened to a logged in session, so Windows did fully boot.

I think the timed out problem is unrelated -- I was getting that yesterday while building some debian boxes -- it always attempted to connect to the ip address of the previous box -- time to reboot!

As for me building my own boxes from iso files -- did you make the machine change to 11 in the .json file or further up the chain somewhere?  I suppose I could just add 

```
"vmware.vmx["virtualhw.version"] = "11"
```

and submit a PR for that?
